polylogue/cli/runs.py

```
@@ -0,0 +1,90 @@
+
+
+def _timestamp_in_range(timestamp: Optional[str], since: Optional[float], until: Optional[float]) -> bool:
+    if timestamp is None:
+        return True
+    try:
+        if timestamp.endswith("Z"):
+            dt = datetime.fromisoformat(timestamp.replace("Z", "+00:00"))
+        else:
+            dt = datetime.fromisoformat(timestamp)
+        epoch = dt.astimezone(timezone.utc).timestamp()
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)